src/dotnet/README.md.j2

```diff
@@ -0,0 +1,176 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+.NET is a general purpose development platform.
+It is cross-platform, and can be used in devices, cloud, and embedded/IoT scenarios.
+You can use C# or F# to write .NET applications.
+
+{% if image.name == "dotnet-sdk" -%}
+This image contains the .NET SDK, which consists of three parts:
+
+- .NET CLI
+- .NET Runtime
+- ASP.NET Core Runtime
+
+Use this image for development, build and test if .NET and ASP.NET Core applications.
+{%- elif image.name == "dotnet-runtime" -%}
+This image contains the .NET runtimes and libraries and is optimized for running .NET applications in production.
+{%- elif image.name == "dotnet-aspnet" %}
+This image contains the ASP.NET Core and .NET runtimes and libraries,
+and is optimized for running ASP.NET Core appplications in production.
+{%- endif %}
+
+## Notice
+
+The .NET packages contained in this image come from a 3rd-party repository at
+[packages.microsoft.com](https://packages.microsoft.com).
+
+You can find the respective source code in [github.com/dotnet](https://github.com/dotnet).
+
+SUSE does not provide any support or warranties for this image.
+
+## Usage
+
+{% if image.name == "dotnet-sdk" -%}
+To compile an application, copy the sources and build the binary:
+
+```Dockerfile
+FROM {{ image.pretty_reference }} AS build
+WORKDIR /source
+
+# copy csproj and restore as distinct layers
+COPY *.csproj .
+RUN dotnet restore
+
+# copy and publish app and libraries
+COPY . .
+RUN dotnet publish --no-restore -o /app
+
+# final stage/image
+FROM registry.suse.com/bci/dotnet-runtime:{{ image.version }}
+WORKDIR /app
+COPY --from=build /app .
+
+# uncomment to run as non-root user
+# USER $APP_UID
+
+ENTRYPOINT ["./dotnetapp"]
+```
+
+Build and run the container image:
+
+```ShellSessionpodmanbuild -t my-dotnet-app .
+podman run -it --rm my-dotnet-app
```

Comment:
```suggestion
```ShellSession
podman build -t my-dotnet-app .
podman run -it --rm my-dotnet-app
```